components/brave_webtorrent/extension/components/torrentViewerHeader.tsx

```diff
@@ -45,6 +52,9 @@ export default class TorrentViewerHeader extends React.PureComponent<Props, {}>
         ? `Start Torrenting ${name}?`
         : 'Loading torrent information...'
     const mainButtonText = torrent ? 'Stop Download' : 'Start Torrent'
+    const copyButtonText = this.props.torrentId.startsWith('magnet:')
+      ? 'Copy Magnet Link'
+      : 'Save Torrent File'
```

Comment:
Please localize these strings

Reply:
This one should also be covered by https://github.com/brave/brave-browser/issues/858.